app/api/resources/mentorship_relation.py

```diff
@@ -821,14 +821,14 @@ class TaskComments(Resource):
     )
     @mentorship_relation_ns.doc(
         responses={
-            HTTPStatus.BAD_REQUEST: f"{messages.UNACCEPTED_STATE_RELATION}",
-            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED}<br>"
-            f"{messages.TOKEN_IS_INVALID}<br>"
-            f"{messages.AUTHORISATION_TOKEN_IS_MISSING}<br>"
-            f"{messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION}",
-            HTTPStatus.NOT_FOUND: f"{messages.USER_DOES_NOT_EXIST}<br>"
-            f"{messages.MENTORSHIP_RELATION_DOES_NOT_EXIST}<br>"
-            f"{messages.TASK_DOES_NOT_EXIST}",
+            HTTPStatus.FORBIDDEN: messages.UNACCEPTED_STATE_RELATION["message"],
+            HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
+            f"{messages.TOKEN_IS_INVALID['message']}<br>"
+            f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}<br>"
+            f"{messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION['message']}",
+            HTTPStatus.NOT_FOUND: f"{messages.USER_DOES_NOT_EXIST['message']}<br>"
+            f"{messages.MENTORSHIP_RELATION_DOES_NOT_EXIST['message']}<br>"
+            f"{messages.TASK_DOES_NOT_EXIST['message']}"
```

Comment:
@Ritika5901234 could you please follow @vj-codes suggestion? The `['message']` would not make the documentation on Swagger UI accurate.